tests/e2e/test_read.py

```diff
@@ -0,0 +1,71 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+import random
+import string
+import pytest
+
+
+from azure.identity import DefaultAzureCredential
+
+from azstoragetorch.io import BlobIO
+
+from azure.storage.blob import BlobServiceClient
```

Comment:
It's a little weird that `ruff` did not remove some of the extra newlines here. I'd recommend installing [`isort`](https://pycqa.github.io/isort/) and running it on this file to organize the imports more to the [pattern](https://peps.python.org/pep-0008/#imports) that is being followed in the repository.